File: Pattern/pattern1/Pattern.java
package Pattern.pattern1;

class Pattern {

    public static void main(String[] args) {
        pattern(5);
    }

    public static void pattern(Integer n) {

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                System.out.print("*");
            }
            System.out.println("");
        }
    }
}

File: Pattern/pattern1/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 0; i < x; i++ {
		for j := 0; j < x; j++ {
			fmt.Print("*")
		}
		fmt.Println("")
	}
}

func main() {
	pattern(5)
}
